notebooks/Making a Custom Statespace Model.ipynb

```diff
@@ -0,0 +1,1464 @@
+{
```

Comment:
<p>I suggest doing a try/expect and printing the error in the except block. This way the notebook can be run all at once without having to resume after the error</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-experimental/pull/174/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>